packages/cli/src/util/parse-arguments.ts

```diff
@@ -0,0 +1,29 @@
+import arg from 'arg';
+import getCommonArgs from './arg-common';
+
+type ParserOptions = {
+  permissive?: boolean;
+};
+
+type Handler = (value: string) => any;
+
+interface Spec {
+  [key: string]: string | Handler | [Handler];
+}
+
+export default function parseArguments<T extends Spec>(
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏make it clear this replaces `getArgs`

I think we should move this new function to live in `packages/cli/src/util/get-args.ts` alongside `getArgs`. I want to make it clear that the code is in the middle of a migration from `getArgs` to `parseArguments`.

I would:

- move `parseArguments` into `packages/cli/src/util/get-args.ts`
- add jsdoc comments to both functions
  - `parseArguments`: explain what it does, returns, and how it replaces `getArgs`
  - `getArgs`: mark as deprecated
-

Reply:
I added a JSDoc comment, but I've never worked with JSDoc before, so please check it over.